code/game/machinery/abominationmachine.dm

```diff
@@ -0,0 +1,179 @@
+////////////////////////////////////////
+//Monster beacon
+////////////////////////////////////////
+
+/obj/item/sbeacondrop/abomachine
+	desc = "A label on it reads: <i>Warning: Activating this device will send a power draining device to your location</i>."
+	droptype = /obj/machinery/abomachine
+	
+////////////////////////////////////////
+//Actual Machine
+////////////////////////////////////////
+	
+/obj/machinery/abomachine
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_g"
+	name = "syndicate bomb"
```

Comment:
Should it be named this?

Reply:
Nope. Haven't touched that. Will do it before finishing touches.